android/app/src/main/java/com/woowacourse/ody/presentation/common/binding/BindingActivity.kt

```diff
@@ -0,0 +1,28 @@
+package com.woowacourse.ody.presentation.common.binding
+
+import android.os.Bundle
+import androidx.annotation.LayoutRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import com.google.android.material.snackbar.Snackbar
+import com.woowacourse.ody.OdyApplication
+
+abstract class BindingActivity<T : ViewDataBinding>(
+    @LayoutRes private val layoutRes: Int,
+) : AppCompatActivity() {
+    protected lateinit var binding: T
```

Comment:
```suggestion
    protected val  binding: T by lazy { DataBindingUtil.setContentView(this, layoutRes) }
```
이 부분 by lazy로 초기화해주면 안 되나요?? 궁금..

Reply:
이 방법이 더 깔끔하네요 변경하겠습니다.